tests/components/nest/test_camera_sdm.py

```diff
@@ -565,10 +573,8 @@ async def test_camera_web_rtc(hass, auth, hass_ws_client):
     assert msg["result"]["answer"] == "v=0\r\ns=-\r\n"
 
     # Nest WebRTC cameras return a placeholder
-    content = await async_get_image(hass)
-    assert content.content_type == "image/jpeg"
-    content = await async_get_image(hass, width=1024, height=768)
-    assert content.content_type == "image/jpeg"
+    assert_placeholder_image(await async_get_image(hass))
+    assert_placeholder_image(await async_get_image(hass, width=1024, height=768))
```

Comment:
I guess it does check whether the call still works with the extra parameters, but it seems like the second assertion isn't really necessary.

Reply:
Removed.